config/openiotsdk/cmake/chip.cmake

```diff
@@ -74,7 +74,6 @@ function(chip_add_data_model target scope model_name)
     include(${CHIP_ROOT}/src/app/chip_data_model.cmake)
     chip_configure_data_model(${target}
         SCOPE ${scope}
-        INCLUDE_SERVER
```

Comment:
Hmm. This argument was meant to resemble `is_server` argument from `chip_data_model.gni`. Removing it is fine as long as there are no out-of-tree controllers built with CMake (I am not aware of any but thought it's worth pointing it out).

Reply:
Thinking that this is not used currently based on examples after several iterations.
If needed, we could revert however if the code path is never tested in any examples (even if only compilation) I have no confidence in us maintaining it.

Willing to add it back (maybe with switched defaults) if someone using cmake is requiring this.